src/pages/WalletManagement/WalletWordsPage.tsx

```diff
@@ -64,32 +63,30 @@ const WalletWordsPage = () => {
   }
 
   return (
-    <MainPanel enforceMinHeight>
-      <PanelContainer>
-        <PanelTitle color="primary" onBackButtonPress={onButtonBack}>
-          Your Wallet
-        </PanelTitle>
-        <PanelContent>
-          <PublicAddressContent>
-            <InfoBox text={plainWallet?.address || ''} label={'Your address'} onClick={handleAddressClick} wordBreak />
-          </PublicAddressContent>
-          <WordsContent inList>
-            <Label>Secret phrase</Label>
-            <PhraseBox>{renderFormatedMnemonic(mnemonic)}</PhraseBox>
-            <InfoBox
-              text={'Carefully note the 24 words. They are the keys to your wallet.'}
-              Icon={Edit3}
-              importance="alert"
-            />
-          </WordsContent>
-        </PanelContent>
-        <FooterActions apparitionDelay={0.3}>
-          <Button onClick={onButtonNext} submit>
-            {"I've copied the words, continue"}
-          </Button>
-        </FooterActions>
-      </PanelContainer>
-    </MainPanel>
+    <FloatingPanel enforceMinHeight>
+      <PanelTitle color="primary" onBackButtonPress={onButtonBack}>
+        Your Wallet
+      </PanelTitle>
+      <PanelContentContainer>
+        <PublicAddressContent>
+          <InfoBox text={plainWallet?.address || ''} label={'Your address'} onClick={handleAddressClick} wordBreak />
+        </PublicAddressContent>
+        <WordsContent inList>
+          <Label>Secret phrase</Label>
+          <PhraseBox>{renderFormatedMnemonic(mnemonic)}</PhraseBox>
+          <InfoBox
+            text={'Carefully note the 24 words. They are the keys to your wallet.'}
```

Comment:
Same as above comment (also, maybe we could possibly DRY this one out?)

Reply:
We may need to use a different wording in both places in the future. I prefer to keep them separated for now.